### ISSUE TITLE
"remember me" checkbox

### ISSUE THREAD

**Issue by OlliTietavainenVaadin** [vaadin/vaadin-login#87]
Missing feature: "Remember me" checkbox, which you could check if you want your login info to be stored so that you don't need to log in again with the same browser unless you log out or some amount of time passes.

**Comment by ugurdoksan** [comment]
I agree with @OlliTietavainenVaadin

**Comment by osenbg** [comment]
Will this be implemented? 
We missing that feature

**Comment by Kleanthismits** [comment]
I can't wait for an out of the box solution for a 'remember me' button. That is a must feature for modern apps.

**Comment by jason-unsworth** [comment]
I agree that this would be a highly appreciated feature for Vaadin's login component.

**Comment by Suplab-commits** [comment]
This feature is very much needed. What is a temporary workaround? Only the Mozilla browser remembers your username and password at the moment and autofill fields.

**Comment by timomeinen** [comment]
@Suplab-commits If you use Spring Security you could enable it by default:

    http.rememberMe().alwaysRemember(true)

**Comment by Suplab-commits** [comment]
@timomeinen Hello. I've tried a million ways, including this one. But if the user is not active for some time, then the session ends anyway, although the validity of the token I set is one year. Why is this happening? All settings work in Security Configuration, except "remember Me". Here is my "Security Config" class. Please help me.
[SecurityConfiguration.zip](https://github.com/vaadin/vaadin-login/files/6386299/SecurityConfiguration.zip)

**Comment by timomeinen** [comment]
@Suplab-commits This is no place for Spring Security support. Please have a look at the [docs](https://docs.spring.io/spring-security/site/docs/current/reference/html5/#servlet-rememberme).

But from what I see in your example, remove the `rememberMeServices` bean. Your `tokenRepository` is sufficient. This should do it:
```
http.rememberMe()
            .alwaysRemember(true)
            .tokenRepository(tokenRepository())
            .and()
```
